cli/command/service/list_test.go

```diff
@@ -0,0 +1,34 @@
+package service
+
+import (
+	"bytes"
+	"testing"
+
+	"golang.org/x/net/context"
+
+	"github.com/docker/cli/cli/internal/test"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/pkg/testutil"
+	"github.com/moby/moby/pkg/testutil/golden"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestServiceListOrder(t *testing.T) {
+	buf := new(bytes.Buffer)
+	cli := test.NewFakeCliWithOutput(&fakeClient{
```

Comment:
`NewFakeCliWithOutput` is deprecated. Please use `NewFakeCli()`. It defaults to an output buffer.

You can get the buffer from `cli.OutBuffer()`